perfact/zodbsync/tests/test_sync.py

```diff
@@ -1105,7 +1105,17 @@ def test_ordered_folder_playback(self):
         self.run('record', '/')
 
         folder = self.repo.path + '/__root__/Test/'
-        os.mkdir(folder + 'new')
+
+        new_folder = folder + 'new'
+        os.mkdir(new_folder)
+
+        with open(os.path.join(new_folder, '__meta__'), 'w') as f:
+            f.write(zodbsync.mod_format({
+                "contents": [],
```

Comment:
Why do you include the `contents`? These are only needed for ordered folders.

Reply:
> Why do you include the `contents`? These are only needed for ordered folders.

Well, because i copied from a few lines below without checking ... fixed